### ISSUE TITLE
[Filter Panel]: QA: Screen reader is identifying Frame for the Sub-options under the  Technologies and Business needs

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9393]
### Description

Page Element:
Sub-options under the  Technologies and Business needs

Issue Description:
In tab/shift tab navigation Screen reader is identifying Frame for the mentioned options

Expected:
Any redundant, excessively wordy and extraneous announcement can be distracting for the screen reader users. 

### Component(s) impacted

Filter Panel

### Browser

Firefox

### Carbon for IBM.com version

v1.38.0-rc.0

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

Web component Canary- https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components Web component Staging- https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by andy-blum** [comment]
@gopinathan-n would you be able to provide a screen recording of this? Based on the browser being firefox, I assume the SR is JAWS?

I've been unsuccessful in replicating this in safari & voiceover, so I'm unsure on how to start fixing this issue.

**Comment by andy-blum** [comment]
Closing in favor of #10589
